proteobench/io/params/__init__.py

```diff
@@ -2,29 +2,30 @@
 from typing import Optional
 
 
+# Reference for parameter names
+# https://github.com/bigbio/proteomics-sample-metadata/blob/master/sdrf-proteomics/assets/param2sdrf.yml
+# new name as comment
 @dataclass
 class ProteoBenchParameters:
     search_engine: Optional[str] = None
     software_version: Optional[str] = None
-    fdr_psm: Optional[str] = None
-    fdr_peptide: Optional[str] = None
-    fdr_protein: Optional[str] = None
-    MBR: Optional[str] = None
-    precursor_tol: Optional[str] = None
+    fdr_psm: Optional[str] = None  # ident_fdr_psm
+    fdr_peptide: Optional[str] = None  # ident_fdr_peptide
+    fdr_protein: Optional[str] = None  # ident_fdr_protein
+    MBR: Optional[str] = None  # enable_match_between_runs
+    precursor_tol: Optional[
+        str
+    ] = None  # precursor_mass_tolerance, value and unit not separated
     precursor_tol_unit: Optional[str] = None
```

Comment:
so we should remove this one if the value is in precursor_mass_tolerance, right?

Reply:
`precursor_tol` and `precursor_tol_unit` will become `precursor_mass_tolerance` - yes